py/desispec/scripts/stdstars.py

```diff
@@ -235,7 +242,10 @@ def main(args, comm=None) :
             # Set fibermask flagged spectra to have 0 flux and variance
             frame = get_fiberbitmasked_frame(frame,bitmask='stdstars',ivar_framemask=True)
             frame_fibermap = frame.fibermap
-            frame_starindices = np.where(isStdStar(frame_fibermap))[0]
+            if std_targetids is None:
+                frame_starindices = np.where(isStdStar(frame_fibermap))[0]
+            else:
+                frame_starindices = np.nonzero(np.in1d(frame_fibermap['TARGETID'], std_targetids))[0]
```

Comment:
Minor: I recently learned that numpy docs say "We recommend using :func:`isin` instead of `in1d` for new code."  They are equivalent for this case.

Reply:
will fix